promregistry/registry.go

```diff
@@ -0,0 +1,31 @@
+package promregistry
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// TeeRegisterer supports MultipleRegisterer.
```

Comment:
Nit: `MultipleRegisterer` looks like there is a type with that name. I would just write plain English: "TeeRegesterer supports multiple Registerers."

Or explain the functionality even a bit more: "TeeRegisterer is a slice of Registerers. It implements Registerer itself and forwards all registrations to all Registerers in the slice."